webdriver/tests/bidi/script/add_preload_script/arguments.py

```diff
@@ -72,7 +72,7 @@ async def test_channel(
 
     on_script_message = wait_for_event("script.message")
     await add_preload_script(
-        function_declaration="""(channel) => channel({'foo': 'bar', 'baz': {'1': 2}})""",
+        function_declaration="""(channel) => channel({'foo': 'bar', 'baz': {'qux': 2}})""",
```

Comment:
Same question here. Why? JavaScript property names are strings. It shouldn’t matter whether that string is `'42'` or `'qux'`, so what’s the motivation for changing this test?

Reply:
They are moved to dedicated serialization tests.